frontend-cert/css-projects/business-card/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Digital Business Card</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div class="business-card">
+        <img src="flower.jpg" alt="Profile Image" class="profile-image">
+        <h1>Your Name</h1>
+        <h2>Software Developer</h2>
+        <h3>@freeCodeCamp</h3>
```

Comment:
```suggestion
        <p>Software Developer</p>
        <p>@freeCodeCamp</p>
```

These are not acting as headings and thus should not be marked up as headings. These are describing where the person works and what their job title is. If you followed the same pattern as you use for email, phone, and portfolio,  you would have:

```
Title: Software Developer
Employer: freeCodeCamp
```

In this case, neither of them would be headings. You are just leaving out the label in front of each one, which is fine, but the lack of labels doesn't make them headings.